tasks/compile.ts

```diff
@@ -4,17 +4,12 @@ var Builder = require('systemjs-builder');
 var util = require('gulp-util');
 var ts = require('gulp-typescript');
 
+var tsProject = ts.createProject('tsconfig.json');
```

Comment:
we can also create different projects per environment for example `tsconfig.test.json`